leakcanary-android/src/main/java/com/squareup/leakcanary/internal/DisplayLeakActivity.java

```diff
@@ -334,12 +334,12 @@ public void onItemClick(AdapterView<?> parent, View view, int position, long id)
                 android.R.drawable.ic_dialog_alert)
                 .setTitle(R.string.leak_canary_delete_all)
                 .setMessage(R.string.leak_canary_delete_all_leaks_title)
-                .setPositiveButton(android.R.string.yes, new DialogInterface.OnClickListener() {
+                .setPositiveButton(android.R.string.ok, new DialogInterface.OnClickListener() {
                   @Override public void onClick(DialogInterface dialog, int which) {
                     deleteAllLeaks();
                   }
                 })
-                .setNegativeButton(android.R.string.no, null)
+                .setNegativeButton(android.R.string.cancel, null)
```

Comment:
these are the same strings on the platform (always ok and cancel), so Lint warns about the misleading names.